### PR TITLE
Support the repackage of eimage and add the related test cases.

### PR DESCRIPTION
**Pull request by teweitsai**
Support the repackage of eimage and add the related test cases.

### CODE REVIEW

File: bin.src/phosim_repackager.py [hunk 1]
```diff
@@ -6,8 +6,10 @@
 import argparse
 import lsst.phosim.utils as phosim_utils
 
-parser = argparse.ArgumentParser(description="Repackager for phosim amp files")
+parser = argparse.ArgumentParser(description="Repackager for phosim files")
```
Comment (SimonKrughoff) [inline bin.src/phosim_repackager.py]: Maybe indicate that the default is to repackage amp files.

File: python/lsst/phosim/utils/phosim_repackager.py [hunk 2]
```diff
@@ -199,3 +201,118 @@ def mef_filename(phosim_amp_file, out_dir='.'):
         if outfile.endswith('.gz'):
             outfile = outfile[:-len('.gz')]
         return outfile
+
+    def process_visit_eimg(self, visit_dir, out_dir=None, prefix='lsst',
```
Comment (SimonKrughoff) [inline python/lsst/phosim/utils/phosim_repackager.py]: The method name is already pretty long.  I'd think about calling it `process_visit_eimage` just to be explicit.  Same suggestion for methods below.

File: python/lsst/phosim/utils/phosim_repackager.py [hunk 3]
```diff
@@ -199,3 +201,118 @@ def mef_filename(phosim_amp_file, out_dir='.'):
...
+        Parameters
+        ----------
+        phosim_eimg_file : str
+            PhoSim eimage file.
+        out_dir : str, optional
+            Output directory (the default is '.'.)
+        """
+
+        # Extract the data for each segment from the FITS files
+        # into a dictionary keyed by channel id.
```
Comment (SimonKrughoff) [inline python/lsst/phosim/utils/phosim_repackager.py]: This comment doesn't seem to line up with this code.

**Review by SimonKrughoff — APPROVED** [review]
Looks good.  Thanks for the test.

File: bin.src/phosim_repackager.py (hunk 1 above)
Comment (teweitsai) [reply bin.src/phosim_repackager.py]: Corrects in the new commit: edbbf54.

File: python/lsst/phosim/utils/phosim_repackager.py (hunk 2 above)
Comment (teweitsai) [reply python/lsst/phosim/utils/phosim_repackager.py]: Corrects in the new commit: edbbf54.

File: python/lsst/phosim/utils/phosim_repackager.py (hunk 3 above)
Comment (teweitsai) [reply python/lsst/phosim/utils/phosim_repackager.py]: Corrects in the new commit: edbbf54.

**Pull request merged**